server/migrations/13_transactions.up.sql

```diff
@@ -0,0 +1,23 @@
+BEGIN;
+
+CREATE TABLE transactions (
+    id VARCHAR NOT NULL,
+    version integer NOT NULL,
+    name varchar,
+    description varchar,
+    created_at timestamp,
+
+    CONSTRAINT transaction_pkey PRIMARY KEY (id, version)
+);
+
+CREATE TABLE transaction_steps (
+    transaction_id varchar NOT NULL,
+    transaction_version integer NOT NULL,
+    test_id varchar NOT NULL,
+    step_number int NOT NULL,
+
+    CONSTRAINT transaction_steps_transactions_fk
+      FOREIGN KEY (transaction_id, transaction_version) REFERENCES transactions(id, "version")
```

Comment:
don't you need an fk here to the tests table?

Reply:
Thing is that the PK of tests are ìd` and `version`. But as we discussed a couple of days ago, we are always going to use the latest version available. So I decided to not keep the fk here